### DISCUSSION TITLE
如何只下载视频文件而不创建 [视频名称 - bilibili] 这个目录？

### DISCUSSION THREAD

**Discussion by ouuan** [yutto-dev/bilili#83] [Q&A] [answered]
现在下载时会创建一个 [视频名称 - bilibili] 目录，视频在其下的 Videos 目录中。我个人在大多数情况下只需要视频文件，不需要弹幕之类的其它东西，而且希望直接放在当前目录下而不是放在某个子目录下。不知道当前能否实现，或者能否增加这个功能。

**Comment by SigureMo** [comment, marked as answer]
很抱歉，v1 中暂时没有相关的 Option，也正如 https://github.com/SigureMo/bilili/issues/76#issuecomment-775901321 中所述，我对现阶段的整体设计都很不满意，如果有 v2 我一定会考虑这一点的 (・ω< )★
